minvws/nl-kat-coordination
# Add json download to report export

### Changes
- When viewing the report we fetch the Report OOI from the id in url.
- The report OOI has the bytes id which we can fetch the report data from bytes.
- Concatenated reports needs further extraction from bytes as we have to fetch each child report data.
- Dump this data in the json export function.
- Did also some changes regarding the way templates get data, make it more consistent.


### Issue link
#3366

Closes #3366 

### QA notes

1. Test JSON download for Aggregate and Generate report  (as concatenated and single report).
2. Check if data downloaded from json matches report data from the HTML report.

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
